### DISCUSSION TITLE
File length too long when cloning to VSCode

### DISCUSSION THREAD

**Discussion by jag-sandhu** [Azure/AzOps#616] [Q&A] [answered]
Hi all,
Has anyone else come across file length too long when cloning to an IDE?
Seems to be mainly microsoft.authorization_roleassignments-xxx files

Any ideas on how to get around this? 

Thanks 
Jag

**Comment by StefanIvemo** [comment, marked as answer]
Have you enabled long file paths in Windows? 

https://github.com/azure/azops/wiki/troubleshooting#windows

**Reply by jag-sandhu** [reply]
Perfect - thanks for pointing this out

**Comment by BenjaminEngeset** [comment]
Tip:

I highly recommend to create a .gitconfig file at the root of the repo so each individual don't have to configure this in their IDE's.
```
[core]
longpaths = true

# Support for long paths on Windows:
# https://docs.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=powershell#enable-long-paths-in-windows-10-version-1607-and-later
```
